build.py

```diff
@@ -297,7 +297,7 @@ def build_variable_fonts(designspace, *steps):
     if args.static_fonts == True:
 
         print ("*** *** *** Autohinting Static Fonts *** *** ***")
-        otfs = list(Path("build").glob("*.otf"))
+        otfs = list(Path("build/static").glob("*.otf"))
```

Comment:
@aaronbell i think this is correct -- we wanna autohint the statics, not the variables :smile:

Reply:
Ah yes, that is correct :).